invio/docker-compose.yml

```
@@ -0,0 +1,33 @@
+services:
+
+  app_proxy:
+    environment:
+      APP_HOST: invio_frontend_1
+      APP_PORT: 8000
+      PROXY_AUTH_ADD: "false"
+
+  backend:
+    image: ghcr.io/kittendevv/invio-backend:v1.7.0@sha256:1d7bc45a698e6ee753e34bbb8e7d1d56c8afc338a6cca8f025894667130555ed
+    environment:
+      ADMIN_USER: "admin"
+      ADMIN_PASS: ${APP_PASSWORD}
+      JWT_SECRET: ${APP_SEED}
+      SESSION_TTL_SECONDS: 43200
+      SECURE_HEADERS_DISABLED: false
+      ENABLE_HSTS: true
+      DATABASE_PATH: ${APP_DATA_DIR}/data/db:/app/data/invio.db
+    volumes:
+      - ${APP_DATA_DIR}/data/app:/app/data
+    restart: on-failure
+
+  frontend:
+    image: ghcr.io/kittendevv/invio-frontend:v1.7.0@sha256:e0c63bbeb4c1c7a0ce80c07b185447d58740a8fc882ebc35a74f7426f6317d2d
+    environment:
+      PORT: 8000
+      BACKEND_URL: http://backend:3000
+      COOKIE_SECURE: false
+      FRONTEND_SECURE_HEADERS_DISABLED: false
+      ENABLE_HSTS: false
```

fix the linter error of invalid type.
```suggestion
      ENABLE_HSTS: "false"
```